src/components/Molecules/ContentsLive.tsx

```diff
@@ -42,13 +43,71 @@ const LiveContents = () => {
         )
           .then((res) => res.json())
           .then((res) => res.contents)
-        const sortData = fetchDate.sort(compare).slice(0, 5)
-        setArticlesData(sortData)
-        setActiveLive(sortData[0])
+        const currentDate = new Date()
+        // 現在の日付と比較して直近の2日間のarticlesDataを取得
+        const filteredData = fetchDate.filter(function (value: Article) {
+          console.log(fetchDate)
+          const year = Number(value.date.slice(0, 4))
+          const month = Number(value.date.slice(5, 7))
+          const day = Number(value.date.slice(8, 10))
+
+          const eachDate = new Date(year, month - 1, day)
+
+          if (currentDate <= eachDate) {
+            return value
+          }
+        })
+
+        let twoDaysDataToDisplay
+        if (filteredData.length >= 2) {
+          twoDaysDataToDisplay = filteredData
+            .sort(compareByAscendingOrder)
+            .slice(0, 2)
+        }
```

Comment:
例えばここは、compareの不等号を逆にしなくてもこんな感じでいけるはず
```suggestion
        if (filteredData.length >= 2) {
          const hoge = filteredData.sort(compareByAscendingOrder).slice(-2)
          setArticlesData(hoge)
        }
```

Reply:
なるほど！！
sliceってマイナス指定で配列の後ろから取得できるんですね！！さっすが！！👍
じゃあこんな感じはどうですか？？

`if (filteredData.length >= 2) {
          twoDaysDataToDisplay = filteredData.slice(-1)
          twoDaysDataToDisplay.push(filteredData[filteredData.length - 2])
        }`

comapre自体使わなくてよいので読みやすいかと！